calico-cloud/tutorials/calico-cloud-features/tour.mdx

```diff
@@ -18,10 +18,10 @@ Let's go through each item in the Manager UI left navbar from top to bottom. You
 The Dashboard provides a birds-eye view of cluster activity. Note the following:
 
 - The filter panel at the top lets you change dashboard views and the time range.
-- The **Layout Settings** shows the default metrics. To get WireGuard metrics for pod-to-pod and host-to-host encryption, you must [enable WireGuard](../../compliance/encrypt-cluster-pod-traffic.mdx).
+- The **Customize Layout** shows the default metrics. To get WireGuard metrics for pod-to-pod and host-to-host encryption, you must [enable WireGuard](../../compliance/encrypt-cluster-pod-traffic.mdx).
```

Comment:
Can we fix this line? 

Perhaps something more like:

The **Customize Layout** menu lets you choose what components are displayed on the dashboard.

Reply:
Needs SME review